### ISSUE TITLE
[Bug]: Can't install on secondary SSD--seemingly random errors

### ISSUE THREAD

**Issue by roblaughter** [Comfy-Org/desktop#579]
### App Version

0.4.5, 0.4.4, 0.4.3

### Expected Behavior

The application should install to D:\ComfyUI.

### Actual Behavior

The application does not install when using a secondary SSD D:\ComfyUI. Instead, I get an "Unable to Start Comfy Desktop" error on every attempt. I've tried 0.4.5, 0.4.4, and 0.4.3. No version will install. 

Each time, I get some random crash on a random package. I've tried completely removing the system files from both AppData locations, deleting the installation from the target directory, and starting fresh multiple times. 

One attempt: 

```
[2024-12-28 00:08:29.713] [info]    x Failed to download `filelock==3.13.1`
  |-> Failed to read from the distribution cache
  `-> Incorrect function. (os error 1)
  help: `filelock` (v3.13.1) was included because `torch` (v2.5.1+cu121)        
        depends on `filelock`
PS D:\ComfyUI> echo "_-end-1735362507218:$?"
```

Another attempt: 

```
[2024-12-28 00:08:31.851] [info]    x Failed to download `trampoline==0.1.2`
  |-> Failed to read from the distribution cache
  `-> Incorrect function. (os error 1)
  help: `trampoline` (v0.1.2) was included because `torchsde` (v0.2.6)
        depends on `trampoline`
```

And a third time: 

```
[2024-12-28 00:15:05.446] [info]    x Failed to download `typing-extensions==4.12.2`
  |-> Failed to read from the distribution cache
  `-> Incorrect function. (os error 1)
  help: `typing-extensions` (v4.12.2) was included because `spandrel`
        (v0.4.0) depends on `typing-extensions`
```

### Steps to Reproduce

1. Run installer.
2. Choose NVIDIA with Cuda.
3. Set path to D:\ComfyUI
4. Begin installation. 
5. Installation crashes with "Unable to Start Comfy Desktop v0.4.x.
6. Click "Open Logs"
7. View error.

### Debug Logs

```powershell
N/A
```

### Browser Logs

N/A

### Setting JSON

N/A

**Comment by robinjhuang** [comment]
@roblaughter Can you upload the full logs (main.log)? [Instructions here ](https://comfyorg.notion.site/ComfyUI-Desktop-User-Guide-1146d73d365080a49058e8d629772f0a#1486d73d365080999843ea13fda2fac1)

**Comment by roblaughter** [comment]
@robinjhuang no, because I completely deleted the broken install to start from scratch and install to the default location. I realize that isn’t helpful.

**Comment by robinjhuang** [comment]
I see, so it worked on your C drive but not external SSD? We've seen this before and can look into it

**Comment by robinjhuang** [comment]
![CleanShot_2024-12-31_at_19 40 512x](https://github.com/user-attachments/assets/8bfb9ca8-03f4-4a1f-8ab8-1c86cf29ee81)

Reproduced.
